### PR TITLE
Idr stream module refactor

### PR DESCRIPTION
**Pull request by roshankern**
This PR is ready for review!

The main focus of this PR is extracting CP features for the MitoCheck labeled dataset (in addition to the DP features that have already been extracted). 

The structure of this module has also been shifted to utilize the different conda environments for `idrstream_cp` and `idr_stream_dp`. Instead of using 3 python files for the different data types (training, negative control, positive control), this PR uses 3 python files for the CP feature extraction, DP feature extraction, and merging. In the CP/DP feature extraction files, for loops are used to extract the 3 different data types (training, negative control, positive control). The structure shift is shown in the screenshots below:

Before PR:
image

After PR:
image

### CODE REVIEW

File: 1.idr_streams/idr_streams.sh [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+conda activate idrstream_cp
+# extract cp features
+python cp_streams.py
+
+# activate conda environment for idrstream dp
+conda activate idrstream_dp
+# extract cp features
+python dp_streams.py
+
+# merge features
```
Comment (jenna-tomkinson) [inline 1.idr_streams/idr_streams.sh:16]: Do you need to activate a specific conda environment to perform the merge features or is it fine that the idrstream_dp environment is activated?

Comment (jenna-tomkinson) [reply 1.idr_streams/idr_streams.sh:16]: I just saw you don't activate an environment for the merge features, but you have separate environments for CP and DP (of course) so I wanted to make sure that there isn't a conda environment for this step.

**Review by jenna-tomkinson — APPROVED** [review]
LGTM! I just asked one clarifying question but other than that this is a very clean PR!

File: 1.idr_streams/idr_streams.sh (hunk 1 above)
Comment (roshankern) [reply 1.idr_streams/idr_streams.sh:16]: Thanks @jenna-tomkinson! A separate environment is not needed for this step as the `merge_streams.py` file only uses `pandas` and `pathlib` and thus can be used with either environment (after `dp_streams.py` the `idrstream_dp` environment will be used)

Comment (jenna-tomkinson) [reply 1.idr_streams/idr_streams.sh:16]: Perfect! Thank you for the clarification! 😄

**Pull request merged**